solutions/sol18.py

```diff
@@ -0,0 +1,17 @@
+#Himanshu Nachane (@Himanshu1495)

```

Comment:
@songz, I tried keeping it **18.py** but then I found out that Python files should not start with a number for the exact same reason. It would be good to keep the file name as **sol18.py** or anything which starts with a string.